browser/ui/brave_browser_command_controller.cc

```diff
@@ -219,15 +226,29 @@ void BraveBrowserCommandController::InitBraveCommandState() {
   UpdateCommandForWebcompatReporter();
 #if BUILDFLAG(ENABLE_TOR)
   UpdateCommandForTor();
+#endif
+#if BUILDFLAG(ENABLE_AI_CHAT)
+  UpdateCommandForAIChat();
 #endif
   UpdateCommandForSidebar();
   UpdateCommandForBraveVPN();
   UpdateCommandForPlaylist();
   UpdateCommandForWaybackMachine();
+#if BUILDFLAG(ENABLE_AI_CHAT) || BUILDFLAG(ENABLE_AI_CHAT)
```

Comment:
nit:
```suggestion
#if BUILDFLAG(ENABLE_BRAVE_VPN) || BUILDFLAG(ENABLE_AI_CHAT)
```

should we move this up so we can merge all the other `#if BUILDFLAG(ENABLE_AI_CHAT)`s in this file?